src/main/java/uk/gov/ch/service/corporatebody/impl/CorporateBodyServiceImpl.java

```diff
@@ -71,4 +82,24 @@ public CompanyProfileApi getCompanyProfile(String companyNumber) throws Corporat
             throw new CompanyProfileMappingException("Json Processing exception for " + companyNumber);
         }
     }
+
+    @Override
+    public RegisteredEmailAddressJson getRegisteredEmailAddress(String companyNumber) throws CorporateBodyDetailsEmailAddressNotFoundException {
+
+        DataMap dataMap = new DataMap.Builder().companyNumber(companyNumber).build();
+
+        LOGGER.infoContext(companyNumber, "Calling database during Update process to retrieve registered email address for company " + companyNumber, dataMap.getLogMap());
```

Comment:
Appreciate this message was copied from the other service but probably wrong there too, insomuch that the 'during Update process' text should be removed from the log output as the Oracle Query API has no idea of the context in which this method was called in order to retrieve the email address.

Reply:
Corrected both messages (REA & ROE)